manifests/ha/namespace-install.yaml

```diff
@@ -1,4 +1,13613 @@
+                                                            selfHeal:
+                                                              type: boolean
+                                                          type: object
+                                                        retry:
+                                                          properties:
+                                                            backoff:
+                                                              properties:
+                                                                duration:
+                                                                  type: string
+                                                                factor:
+                                                                  format: int64
+                                                                  type: integer
+                                                                maxDuration:
+                                                                  type: string
+                                                              type: object
+                                                            limit:
+                                                              format: int64
+                                                              type: integer
+                                                          type: object
+                                                        syncOptions:
+                                                          items:
+                                                            type: string
+                                                          type: array
+                                                      type: object
+                                                  required:
+                                                  - destination
+                                                  - project
+                                                  - source
+                                                  type: object
+                                              required:
+                                              - metadata
+                                              - spec
+                                              type: object
+                                          type: object
+                                      type: object
+                                    type: array
+                                required:
+                                - generators
+                                type: object
+                              merge:
+                                properties:
+                                  generators:
+                                    items:
+                                      properties:
+                                        clusterDecisionResource:
+                                          properties:
+                                        
```

Comment:
A lot of users install Argo CD into non `argocd` namespace. Please remove hardcoded namespace from applicationset components.